kiali_api.md

```diff
@@ -9094,6 +9095,7 @@ to be removed in 1.21 release.
 | IstioInitContainers | [][ContainerInfo](#container-info)| `[]*ContainerInfo` |  | |  |  |
 | Labels | map of string| `map[string]string` |  | |  |  |
 | Name | string| `string` |  | |  |  |
+| ServiceAccountName | string| `string` |  | |  |  |
```

Comment:
Why is `ServiceAccountName` and `ServiceAccountNames` being added?

Reply:
To be honest, I'm not sure.
CI asked me to update the swagger.json file. I ran the  `make` target to update it and it also updated this kiali_api.md file. So, I committed it.

Anyway, looks like somebody else already added it: https://github.com/kiali/kiali/blob/master/kiali_api.md?plain=1#L10658

Perhaps I simply need to rebase remove this noise from the PR.

Reply:
So, yes! after rebasing, these changes disappeared.